src/hex/compiletime/flow/parser/expr/PropertyParser.hx

```diff
@@ -158,10 +165,11 @@ class PropertyParser
 				propertyVO = new PropertyVO( ident, fieldName, null, type, ref, null, null, constructorVO );
 
 			case _:
-				logger.debug( assigned.expr );
+				assigned.reject('This type of expression cannot be used here: ${assigned.toString()}');
```

Comment:
This change is somewhat unrelated.

Just logging makes execution continue to line 171 which leads to a null reference error that comes with an impressive stack trace, so I think failing here is more practical.